microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -99,27 +103,31 @@ def __init__(
         # tar file path specified in yml
         self._tar_file_path = kselftest_file_path
         if self._tar_file_path:
+            self._remote_tar_path = (
+                self.node.get_pure_path("/tmp/kselftest") / os.path.basename(self._tar_file_path)
```

Comment:
`self.node.get_pure_path("/tmp/kselftest")` called twice, use a variable to share the logic.